README.md

```diff
@@ -47,7 +47,8 @@ This script will create a new Django project.
 ## Development
 
 To run the test suite, run `pipenv install --dev` and then `pipenv run python
--m pytest` from this repository directory
+-m pytest` from this repository directory. 
```

Comment:
**suggestion (non-blocking)**: Since we added the init.py file to be able to run `pytest` directly, let's change this to say just `pipenv run pytest`